baseapp/abci.go

```diff
@@ -842,8 +842,8 @@ func (app *BaseApp) internalFinalizeBlock(ctx context.Context, req *abci.Request
 
 func (app *BaseApp) executeTxs(ctx context.Context, txs [][]byte) ([]*abci.ExecTxResult, error) {
 	if app.txExecutor != nil {
-		return app.txExecutor(ctx, len(txs), app.finalizeBlockState.ms, func(i int, ms storetypes.MultiStore, incarnationCache map[string]any) *abci.ExecTxResult {
-			return app.deliverTxWithMultiStore(txs[i], i, ms, incarnationCache)
+		return app.txExecutor(ctx, len(txs), app.finalizeBlockState.ms, func(i int, ms storetypes.MultiStore) *abci.ExecTxResult {
```

Comment:
txExecutor interface need to keep too.

Reply:
we just [need](https://github.com/crypto-org-chain/ethermint/blob/0e0baa4ea54079296cebf233fe7510700f981830/app/ante/handler_options.go#L116-L122) get and set right? Do you mean keep txExecutor but pass nil?

Reply:
[stm executor](https://github.com/crypto-org-chain/ethermint/blob/develop/app/executor.go#L66) will pass non-nil cache.